### PR TITLE
Dump output of layers for debugging

### PR DESCRIPTION
**Pull request by LakshmiKumar23**
Includes changes from  #144

### CODE REVIEW

**Comment by kiritigowda** [comment]
@LakshmiKumar23 please resolve the conflicts on this PR

**Comment by LakshmiKumar23** [comment]
@kiritigowda Resolved

**Comment by paveltc** [comment]
Can be merged.

**Comment by kiritigowda** [comment]
@LakshmiKumar23 fix the conflicts

**Comment by LakshmiKumar23** [comment]
@kiritigowda fixed conflicts

File: model_compiler/python/caffe_to_nnir.py [hunk 1]
```diff
@@ -309,7 +309,7 @@ def extractCaffeAttrInfo(layer_param):
         variance_encoded_in_target = detection_output.variance_encoded_in_target
         keep_top_k = detection_output.keep_top_k
         confidence_threshold = detection_output.confidence_threshold
-
```
Comment (kiritigowda) [inline model_compiler/python/caffe_to_nnir.py]: Remove this

File: amd_openvx_extensions/amd_nn/src/activation_layer.cpp [hunk 2]
```diff
@@ -36,7 +36,6 @@ struct ActivationLayerLocalData {
 
 };
 
-
```
Comment (kiritigowda) [inline amd_openvx_extensions/amd_nn/src/activation_layer.cpp]: remove

File: README.md [hunk 3]
```diff
@@ -1,308 +0,0 @@
-[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://opensource.org/licenses/MIT)
```
Comment (kiritigowda) [inline README.md]: Add this back

File: amd_openvx/openvx/ago/ago_util.cpp [hunk 4]
```diff
@@ -25,6 +25,9 @@ THE SOFTWARE.
 #include <math.h>
 #include <sstream>
 
+
```
Comment (kiritigowda) [inline amd_openvx/openvx/ago/ago_util.cpp]: Remove extra space

File: amd_openvx/openvx/ago/ago_util.cpp [hunk 5]
```diff
@@ -25,6 +25,9 @@ THE SOFTWARE.
...
+#define ENABLE_DEBUG_DUMP_OPENVX_BUFFERS                      0
+
```
Comment (kiritigowda) [inline amd_openvx/openvx/ago/ago_util.cpp]: remove

File: model_compiler/python/nnir_to_openvx.py [hunk 6]
```diff
@@ -599,9 +608,9 @@ def generateModuleCPP(graph,fileName):
       ERROR_CHECK_STATUS(vxReleaseNode(&node));
     }
 """ % (node.inputs[0], node.outputs[0]))
-            elif node.type == 'copy'or node.type == 'transpose' or node.type == 'permute': 
+            elif node.type == 'copy'or node.type == 'transpose' or node.type == 'permute':  
                 if node.type == 'transpose':
-                    order_list = node.attr.get('axes')
+                    order_list = node.attr.get('axes')
```
Comment (kiritigowda) [inline model_compiler/python/nnir_to_openvx.py]: remove

File: model_compiler/python/nnir_to_openvx.py [hunk 7]
```diff
@@ -599,9 +608,9 @@ def generateModuleCPP(graph,fileName):
       ERROR_CHECK_STATUS(vxReleaseNode(&node));
     }
 """ % (node.inputs[0], node.outputs[0]))
-            elif node.type == 'copy'or node.type == 'transpose' or node.type == 'permute': 
+            elif node.type == 'copy'or node.type == 'transpose' or node.type == 'permute':
```
Comment (kiritigowda) [inline model_compiler/python/nnir_to_openvx.py]: remove

File: amd_openvx/openvx/ago/ago_util.cpp (hunk 5 above)
Comment (LakshmiKumar23) [reply amd_openvx/openvx/ago/ago_util.cpp]: done

File: amd_openvx/openvx/ago/ago_util.cpp (hunk 4 above)
Comment (LakshmiKumar23) [reply amd_openvx/openvx/ago/ago_util.cpp]: done

File: amd_openvx/openvx/ago/ago_util.cpp [hunk 8]
```diff
@@ -20,11 +20,12 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-
 #include "ago_internal.h"
 #include <math.h>
 #include <sstream>
 
+#define ENABLE_DEBUG_DUMP_OPENVX_BUFFERS                      0
```
Comment (kiritigowda) [inline amd_openvx/openvx/ago/ago_util.cpp]: Check

**Pull request merged**